src/apps/backend/modules/comment/internal/store/comment-db.ts

```diff
@@ -0,0 +1,36 @@
+import { Schema, Types } from 'mongoose';
+
+export interface CommentDB {
+  _id: Types.ObjectId;
+  account: Types.ObjectId;
+  content: string;
+  task: Types.ObjectId;
+}
+
+export const CommentDbSchema = new Schema<CommentDB>(
+  {
+    account: {
+      type: Schema.Types.ObjectId,
+      ref: 'accounts',
+      required: true,
+    },
+    content: {
+      type: String,
+      required: true,
+      trim: true,
+      minlength: 1,
+    },
+    task: {
+      type: Schema.Types.ObjectId,
+      ref: 'Task',
+      required: true,
```

Comment:
Do we need to index this?

Can you check CommentReader file and see if we need to create other combined indexes?